docs/middleware/android-sdk.md

```diff
@@ -1,190 +1,126 @@
-# Sourcing Entity - Android SDK
-FinBox Lending SDK is a one stop solution to seamlessly onboard users to a loan application journey.
+# FinBox Lending: Android
 
-## Modules
-The SDK is broken down into multiple modules which can be customizable as per requirements. Complete SDK has following modules:
-1. Onboarding
-2. Loan dashboard
+FinBox Lending SDK is a drop-in module which can add a digital lending journey to any mobile application
+The SDK has following modules:
+
+1. OnBoarding
+2. Loan Dashboard
 3. KYC
-4. Loan offer and agreement
+4. Loan offer and Agreement
+
+## OnBoarding module
+
+Onboarding the user is the first and the most critical step of the SDK. FinBox does the onboarding based on OTP verification of user's mobile number.
+
+## Loan Dashboard
+
+The loan dashboard screen will have details of the active/ongoing loan for that user. This screen gives an overview about the loan journey.
+In case the user has a disbursed loan he can view his EMI details in the Repay section.
```

Comment:
Add another space, if you want this to be a different paragraph. See Deploy Preview.